.travis.yml

```
@@ -1,10 +1,26 @@
 services:
   - docker
 
+branches:
+  only:
+    - master
+    - module-8
+
 script:
   - docker build --target test --tag my-test-image . 
   - docker run my-test-image tests
   - docker run --env SECRET_KEY --env TRELLO_TODOID --env TRELLO_DONEID --env TRELLO_BOARDID --env TRELLO_KEY --env TRELLO_TOKEN my-test-image test_e2e
+before_deploy:
+  - docker login --username $DOCKER_USERNAME --password $DOCKER_PASSWORD
+  - docker build --target production --tag aj1993/my-image-prod:latest .
+  - docker push aj1993/my-image-prod:latest
```

Doesn't really matter, but may as well use the variable here
```suggestion
  - docker build --target production --tag ${DOCKER_USERNAME}/my-image-prod:latest .
  - docker push ${DOCKER_USERNAME}/my-image-prod:latest
```
Same in the deploy script

---

Only did it like that because the instructions said that I could do something like that.

---

But yes I should change it to the variable version

---

Worked. Though shouldn't I do the same for line 20?

---

Yep, worth doing the same in the deploy section